### PR TITLE
gtsam: depend on boost155

### PR DESCRIPTION
**Pull request by ilovezfs**
primarily intended to trigger CI

### CODE REVIEW

**Comment by ilovezfs** [comment]
@sjackman this is the same failure I'm seeing locally.

**Comment by ilovezfs** [comment]
Just confirmed it builds fine with boost155, and that found that it also fails with boost159.

**Comment by ilovezfs** [comment]
Hopefully, there can be an El Capitan boost155 bottle to fix this https://github.com/Homebrew/homebrew-versions/pull/1330.

File: gtsam.rb [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 
   bottle do
     sha256 "5e1040d1a55f395b038e5ee5647377a2b8182f2af0536f8e8c028c81cc7f57d9" => :yosemite
     sha256 "b62f92c94589f4794cfe67076213aeae26234a464aa87727e2312da32c277eb4" => :mavericks
     sha256 "4c9a630f008d37f1ecd19cd1ce87912206f2eed24caf0c7b6404b5c08eba74cf" => :mountain_lion
   end
 
+  option "with-test", "Run unit tests at build-time"
+
+  deprecated_option "without-tests" => "without-test"
```
Comment (sjackman) [inline gtsam.rb]: I think this line should be 
```ruby
  deprecated_option "with-tests" => "with-test"
```

Comment (ilovezfs) [inline gtsam.rb]: fixed.

**Pull request closed without merging**